### DISCUSSION TITLE
Connect to Reports API 400

### DISCUSSION THREAD

**Discussion by madibai** [iTwin/itwinjs-core#3555] [Q&A] [answered]
Hi, i'm new in itwin

Report creating tutorial (https://developer.bentley.com/apis/insights/operations/create-report/) working correctly
<img width="768" alt="Снимок экрана 2022-04-27 в 04 27 13" src="https://user-images.githubusercontent.com/42195915/165404111-9706b311-f974-4bc3-9981-de27d56417f3.png">

Try repeat on Postman same query with params - got error

<img width="927" alt="Снимок экрана 2022-04-27 в 04 38 46" src="https://user-images.githubusercontent.com/42195915/165404274-2af5c6de-650f-404d-a885-9aa66f2381c9.png">

where is my fail?

**Comment by aruniverse** [comment, marked as answer]
In your postman post request, you do not need to put `HTTP/1.1` in the request url. That's just the protocol
